applications/GeoMechanicsApplication/tests/test_integration_node_extrapolation/4_noded_quadrilateral/MaterialParameters.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I gues we are using the same material for all element variables. we can reuse in similar to what Mohammed did for T and Pw element and condition tests.